installer/templates/new/assets/javascripts/application.js

```diff
@@ -2,7 +2,20 @@ import '../stylesheets/application.scss';
 
 import $ from 'jquery';
 import {Kitto} from 'kitto';
+import fscreen from 'fscreen';
 
 window.jQuery = window.$ = $;
 
 Kitto.start();
+
+var i = null;
```

Comment:
application.js is meant for the user to extend / modify appearance and behaviour.

Let's have this in an `initializeFullScreenButton` function in https://github.com/kittoframework/kitto/blob/master/priv/static/kitto.js also move the `fscreen` import there.

Reply:
Something like the following (haven't tried it):

```javascript
function initializeFullScreenButton() {
  var timer;
  var $button = $('.fullscreen-button');

  $('body').on('mousemove', function() {
    clearTimeout(timer);
    if (!$button.hasClass('active')) { $button.addClass('active') }
    timer = setTimeout(function() { $button.removeClass('active') }, 1000);
  })

  $button.on('click', function() {
    fscreen.requestFullscreen(document.getElementById('container'));
  })

  return this;
}
```

which will be called in Kitto.start